tests/save_test.go

```diff
@@ -113,6 +113,15 @@ func TestSaveDoCommand(t *testing.T) {
 		"async":    true,
 	}
 
+	// Invalid async save with future timestamp
+	saveCmd5 := map[string]interface{}{
+		"command":  "save",
+		"from":     "2024-09-06_15-00-33",
+		"to":       "2025-09-06_15-00-33",
```

Comment:
Should we bump to 2030 so test does not break in September?

Reply:
True. Maybe 3030?

Reply:
3024